modules/core/src/v2/coins/tbtc.ts

```diff
@@ -22,4 +22,8 @@ export class Tbtc extends Btc {
   getFullName() {
     return 'Testnet Bitcoin';
   }
+
+  defaultChangeAddressType() {
+    return this.supportsP2wsh() ? `p2wsh` : this.supportsP2shP2wsh() ? `p2shP2wsh` : `p2sh`;
```

Comment:
`return "p2wsh";` would be cleaner here (the commit message suggests that it is equivalent)